convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -52,16 +52,7 @@ def run(self):  # pylint: disable= too-many-return-statements
             "C2R\\t%{BUILDTIME}\\t%{VERSION}-%{RELEASE}\\t%{REPOID}",
         ]
 
-        reposdir = get_hardcoded_repofiles_dir()
-        if reposdir and not system_info.has_internet_access:
-            logger.warning("Did not perform the check as no internet connection has been detected.")
-            self.add_message(
-                level="WARNING",
-                id="IS_LOADED_KERNEL_LATEST_CHECK_SKIP",
-                title="Did not perform the is loaded kernel latest check",
-                description="Did not perform the check as no internet connection has been detected.",
-            )
-            return
+        reposdir = get_backedup_system_repos()
```

Comment:
I think we're trying to access the backup dir too soon, no repositories could have gotten backed up at this point @r0x0d 
I've tried `run_subprocess(["tree", reposdir])` [on this line](https://github.com/oamg/convert2rhel/pull/1181/files#diff-e4898f3961034627f745928957cb0c208e05b836679f68c333915be44b395c11R62), and this is what we get.
```
[2024-04-18T06:59:37-0400] TASK - [Prepare: Check if the loaded kernel version is the most recent] ***
[2024-04-18T06:59:37-0400] DEBUG - Calling command 'tree /var/lib/convert2rhel/backup/ed2e326f3aefc335463ab68aa15a72c2'
/var/lib/convert2rhel/backup/ed2e326f3aefc335463ab68aa15a72c2

0 directories, 0 files
[2024-04-18T06:59:37-0400] DEBUG - Calling command 'repoquery --setopt=exclude= --quiet --qf C2R\t%{BUILDTIME}\t%{VERSION}-%{RELEASE}\t%{REPOID} --setopt=reposdir=/var/lib/convert2rhel/backup/ed2e326f3aefc335463ab68aa15a72c2 kernel-core'
[2024-04-18T06:59:38-0400] DEBUG - Got a line without the C2R identifier: 
ERROR - (OVERRIDABLE) IS_LOADED_KERNEL_LATEST::KERNEL_CURRENCY_CHECK_FAIL - Kernel currency check failed
 Description: Please refer to the diagnosis for further information
 Diagnosis: Could not find any kernel-core from repositories to compare against the loaded kernel.
 Remediations: Please, check if you have any vendor repositories enabled to proceed with the conversion.
If you wish to disregard this message, set the environment variable 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK' to 1.
```

Reply:
fixed!